sycl/test-e2e/Basic/bit_cast/win_host_compile.cpp

```diff
@@ -0,0 +1,7 @@
+
+// REQUIRES: windows
+
+// RUN: %clangxx -fsycl -fsycl-host-compiler=cl -fsycl-host-compiler-options='/std:c++17 /Zc:__cplusplus'  -o %t.out  %p/bit_cast.cpp
```

Comment:
I'm not sure we need this - we should be running the entire test-e2e with host compiler internally...

That said, I'd rather add extra `RUN` lines to the original test via `%if windows %{ ... %}`